puppet_runtime.yaml

```diff
@@ -0,0 +1,2 @@
+name: puppet-runtime*
```

Comment:
@nmburgan I could not find how this artifact is produced.  Can you point me the right direction?

Reply:
:facepalm: I somehow missed https://github.com/OpenVoxProject/puppet-runtime